cmake/AddVSCode.cmake

```diff
@@ -0,0 +1,40 @@
+function(generate_vscode)
```

Comment:
A doc-comment explaining the purpose of this function would be beneficial and appreciated 😃

Reply:
Yes!